lib/sidekiq/prometheus/exporter/metrics.rb

```diff
@@ -23,6 +24,16 @@ def queues_stats
           QueueStats.new(queue.name, queue.size, queue.latency)
         end
       end
+
+      def max_processing_times
```

Comment:
Hey, looks cool, can I suggest those changes?

```ruby
def max_processing_times
  now = Time.now.to_i
  Sidekiq::Workers.new
    .map { |_, _, execution| execution }
    .group_by { |execution| execution['queue'] }
    .each_with_object({}) do |(queue, executions), memo|
      oldest_execution = executions.min_by { |execution| execution['run_at'] }
      memo[queue] = now - oldest_execution['run_at']
    end
end
```

I just change `map.to_h` with `each_with_object` and naming `works` changed to `executions`